pkg/osbuild/osbuild-exec.go

```diff
@@ -144,3 +144,37 @@ func OSBuildVersion() (string, error) {
 	version = strings.TrimSpace(version)
 	return version, nil
 }
+
+// OSBuildInspect converts a manifest to an inspected manifest.
+func OSBuildInspect(manifest []byte) ([]byte, error) {
```

Comment:
Go can help with a lot of this, I think:
```go
func OSBuildInspect(manifest []byte) ([]byte, error) {
	cmd := exec.Command("osbuild", "--inspect")
	cmd.Stdin = bytes.NewBuffer(manifest)
	output, err := cmd.Output()
	if err != nil {
		return nil, err
	}
	return output, nil
}
```
is enough - I did not test though but I'm 80% confident